src/main/webapp/app/job/job-detail/job-detail.component.ts

```diff
@@ -274,4 +285,46 @@ export class JobDetailComponent {
 
     this.jobDetails.set(loadedJob);
   }
+
+  private async loadJobDetailsFromForm(form: JobFormDTO): Promise<void> {
+    const user = this.accountService.loadedUser();
+    let researchGroupDeatils;
+    try {
+      researchGroupDeatils = await firstValueFrom(
+        this.researchGroupService.getRessourceGroupDetails(user?.researchGroup?.researchGroupId ?? ''),
```

Comment:
```suggestion
this.researchGroupService.getRessourceGroupDetails(user?.researchGroup?.researchGroupId ?? ''),
```
Do you need a fallback empty string here? Since research group id should be non-nullable